packages/jobs/lib/runner/render.runner.ts

```diff
@@ -71,11 +71,6 @@ export class RenderRunner implements Runner {
                         { key: 'NODE_ENV', value: process.env['NODE_ENV'] },
                         { key: 'NANGO_CLOUD', value: process.env['NANGO_CLOUD'] },
                         { key: 'NANGO_DB_HOST', value: process.env['NANGO_DB_HOST'] },
-                        { key: 'NANGO_DB_NAME', value: process.env['NANGO_DB_NAME'] },
-                        { key: 'NANGO_DB_PASSWORD', value: process.env['NANGO_DB_PASSWORD'] },
-                        { key: 'NANGO_DB_PORT', value: process.env['NANGO_DB_PORT'] },
-                        { key: 'NANGO_DB_SSL', value: process.env['NANGO_DB_SSL'] },
-                        { key: 'NANGO_ENCRYPTION_KEY', value: process.env['NANGO_ENCRYPTION_KEY'] },
```

Comment:
@khaliqgant can you confirm the runners don't need then encryption key because all potentially encrypted data is either passed already unencrypted via NangoProps or accessed via the node client?

Reply:
Confirmed. Any data being saved goes through the data.service and anything retrieve is decrypted by the service being called before hand